module/modmanager/manager_test.go

```diff
@@ -383,6 +383,12 @@ func TestModManagerValidation(t *testing.T) {
 func TestModuleReloading(t *testing.T) {
 	ctx := context.Background()
 
+	// Lower global timeout early to avoid race with actual restart code.
```

Comment:
[q] How does moving this code to one point at the top of the four tests avoid the race we were seeing with OUEs?

Reply:
the race was related to the goroutine for one test reading oueRestartInterval while the next test was changing it, so this feels like a decent way of avoiding it all together

Reply:
Got it; didn't see it was just about reading the variable; thanks!